extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -27,6 +36,20 @@ public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $ou
 			];
 		}
 
+		if ( isset( $wg->articleVideoRelatedVideos ) ) {
+			$relatedVideo = ArticleVideoController::getRelatedVideoData( $wg->articleVideoRelatedVideos, $title );
+
+			if ( isset( $relatedVideo['videoId'] ) ) {
+				$vars['wgArticleRelatedVideoData'] = [
+					'videoId' => $relatedVideo['videoId'],
+					'playerParams' => [
```

Comment:
I don't think it fits `wgArticleVideoData` as it's not really _a video data_ but rather _the player config_.